### ISSUE TITLE
Table navigation in Libreoffice writer using NVDA not properly working

### ISSUE THREAD

**Issue by soluloquy17** [nvaccess/nvda#7610]
Steps to reproduce:

1. Open Libreoffice Writer
2. Open any document containing at least one table.
3. Press control plus left or right arrow keys, or up or down arrow keys, to navigate tables like in any other commonly used software e.g. Firefox.  
4. Notice that NVDA does not read anything.

Expected behavior:
NVDA should reed column/row texts as it does in other softwares

System configuration:
NVDA version:
2017.3
NVDA Installed or portable:
Installed copy
Windows version:
Windows 10 (64-bit) version 1703 Build 15063 

Name and version of other software in use when reproducing the issue:
Libreoffice - version: 5.4.0.3

**Comment by Brian1Gaff** [comment]
It actually gets stuck and generates errors unless you force your way to the 
next bit of the file in some way I notice.
 I used the  portable version, but I doubt its much different. the error 
mentions Symphony by the way, which shows the legacy of the software, I 
suppose.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.

**Comment by ehollig** [comment]
Duplicate of #1994
Duplicate of #1994 for LibreOffice Writer and #4133 for Open Office Writer
